tests/plugins/easyimage/manual/objecturlerror.html

```diff
@@ -0,0 +1,28 @@
+<div id="classic">
```

Comment:
You included `floatingspace` plugin, but not added inline editor. Please add it.

Reply:
Some leftover 😞